src/Plugin.Maui.Audio/AudioPlayer/AudioPlayer.android.cs

```diff
@@ -32,39 +32,99 @@ public double Balance
 		set => SetVolume(Volume, balance = value);
 	}
 
-	[SupportedOSPlatform("Android23.0")]
-	public double Speed
+	bool isChangingSpeed = false;
+
+	/// <summary>
+	/// Internal state machine for isPlaying. This is needed because when Reset is called, the player is not playing anymore, but in Idle state IsPlaying is not available
+	/// </summary>
+	bool isPlaying = false;
+
+
+	public void SetSpeed(double sp)
```

Comment:
I forget the attribute name but I think we can add it to indicate that only version 23+ is supported. @jfversluis do you know the attribute?

Reply:
I think it's   SupportedOSPlatform

Reply:
Oh I see it was above [SupportedOSPlatform("Android23.9")]

Reply:
Ah you beat me to it :)

Reply:
done